src/components/molecules/UiStepper/_internal/UiStepperStep.vue

```diff
@@ -14,7 +14,7 @@
     >
       <component
         v-bind="itemAttrs"
-        :is="itemTag"
+        :is="UiButton"
```

Comment:
Why we can't use `UiButton` directly? I mean, we don't need `<component />` when we use static `UiButton`